requirements.txt

```diff
@@ -54,7 +55,17 @@ notify2==0.3.1; sys_platform != 'win32' and sys_platform != 'darwin'
 # Uncomment line below or manually install after installing requirements.
 # pygobject>=3.10.2; sys_platform != 'win32' and sys_platform != 'darwin'
 
+# PyYAML does not have binary packages for Mac Users
+PyYAML --no-binary=PyYAML; sys_platform == 'darwin'
+PyYAML; sys_platform != 'darwin'
+
+# Apprise Requirements
+requests
```

Comment:
Please specify the exact versions. It's important for the packaging reproducibility.

Reply:
Done